openpype/tools/launcher/models.py

```diff
@@ -179,11 +183,20 @@ def filter_actions(self):
 
         self.beginResetModel()
 
+        local_settings = get_local_settings()
         items = []
         for order in sorted(items_by_order.keys()):
             for item in items_by_order[order]:
                 item_id = str(uuid.uuid4())
                 item.setData(item_id, ACTION_ID_ROLE)
+
+                if self.is_force_not_open_workfile(item,
```

Comment:
You're calling this even on actions that are not application actions.

Reply:
So `single_actions` are not application actions?

Reply:
ApplicationAction is one type of launcher actions. In launcher can be multiple different actions not related to applications like "Open File Explorer for context".